### PR TITLE
^ outline to include constant & interface

### PR DESCRIPTION
**Pull request by piengeng**
* pending merge by github.com/ramya-rao-a/go-outline, sent pull request at https://github.com/ramya-rao-a/go-outline/pull/10.
* `interface` to follow `struct`'s regex approach
* maybe the SymbolKind for type should be TypeParameter

### CODE REVIEW

**Comment by piengeng** [comment]
what's the minimum command line to run the **/extension.test.ts?

**Comment by ramya-rao-a** [comment]
I usually chose the the test related debug configuration and then run that using F5

File: test/fixtures/outlineTest/test.go [hunk 1]
```diff
@@ -6,6 +6,12 @@ import (
 
 var _ string = "foobar"
 
+// const constFoo = "constBar"
+
+type circle struct {
```
Comment (ramya-rao-a) [inline test/fixtures/outlineTest/test.go]: shouldn't this be interface and not struct?

Comment (piengeng) [reply test/fixtures/outlineTest/test.go]: @ramya-rao-a  you're right, just another stupid mistake. already update to my master.

**Comment by ramya-rao-a** [comment]
The build is failing at the linting step with the below error:

> ERROR: /home/travis/gopath/src/github.com/microsoft/vscode-go/test/integration/extension.test.ts:519:38 - " should be '

**Comment by piengeng** [comment]
I've tried, too complex for me, closing instead.

**Comment by piengeng** [comment]
Thanks, I've looked at the data structures generated by goOutline.ts, the
more I look at it, the more confused I've become. Initially I figure it's
just making behave like the `struct` will do. Anyway, appreciate your help
:)

**Pull request merged**